COLLABORATOR_GUIDE.md

```diff
@@ -515,6 +515,12 @@ Run tests (`make -j4 test` or `vcbuild test`). Even though there was a
 successful continuous integration run, other changes may have landed on master
 since then, so running the tests one last time locally is a good practice.
 
+Validate that the commit message is properly formatted using [core-validate-commit](https://github.com/evanlucas/core-validate-commit).
```

Comment:
Nit: please wrap this into 80 characters per line

Reply:
done